modules/ossm-rn-known-issues.adoc

```diff
@@ -47,19 +51,13 @@ These are the known issues in {ProductName}:
 
 * link:https://issues.jboss.org/browse/MAISTRA-158[MAISTRA-158] Applying multiple gateways referencing the same hostname will cause all gateways to stop functioning.
 
-* link:https://issues.jboss.org/browse/MAISTRA-806[MAISTRA-806] Evicted Istio Operator Pod causes mesh and CNI not to deploy.
-+
-If the `istio-operator` pod is evicted while deploying the control pane, delete the evicted `istio-operator` pod.
+
 
 [id="ossm-rn-known-issues-kiali_{context}"]
 == Kiali known issues
 
 These are the known issues in Kiali:
 
-* link:https://issues.jboss.org/browse/KIALI-3262[KIALI-3262] In the Kiali console, when you click on Distributed Tracing in the navigation or on a Traces tab, you are asked to accept the certificate, and then asked to provide your OpenShift login credentials. This happens due to an issue with how the framework displays the Trace pages in the Console. The Workaround is to open the URL for the Jaeger console in another browser window and log in. Then you can view the embedded tracing pages in the Kiali console.
```

Comment:
So this known issue should remain.

Reply:
Updated to list MAISTRA-873 as a known issue.